### DISCUSSION TITLE
Feature request : maybe MAui can introduce other languages aside from c# and xaml

### DISCUSSION THREAD

**Discussion by Xyncgas** [dotnet/maui#5082] [Ideas]
like flutter, swift, extending the way developers can make ui

**Comment by nogginbox** [comment]
There is [Comet](https://github.com/dotnet/Comet), which is an MVU framework where views are defined using code. It looks interesting, but I've not tried it out and I don't know how feature complete it is.

**Reply by Xyncgas** [reply]
ohh
